flink/v1.12/flink/src/main/java/com/netease/arctic/flink/read/source/log/kafka/LogKafkaSourceBuilder.java

```diff
@@ -69,24 +84,25 @@ public class LogKafkaSourceBuilder {
   private Boundedness boundedness;
   private KafkaRecordDeserializer deserializationSchema;
   // The configurations.
-  protected Properties props;
+  protected Properties kafkaProperties;
 
   private Schema schema;
   private Map<String, String> tableProperties;
 
   /**
    * @param schema          read schema, only contains the selected fields
-   * @param tableProperties arctic table properties with sql hints
+   * @param tableProperties arctic table properties, maybe include Flink SQL hints.
    */
   LogKafkaSourceBuilder(Schema schema, Map<String, String> tableProperties) {
     this.subscriber = null;
     this.startingOffsetsInitializer = OffsetsInitializer.earliest();
     this.stoppingOffsetsInitializer = new NoStoppingOffsetsInitializer();
     this.boundedness = Boundedness.CONTINUOUS_UNBOUNDED;
     this.deserializationSchema = null;
-    this.props = new Properties();
+    this.kafkaProperties = getLogStoreProperties(tableProperties);
```

Comment:
It's better to rename `getLogStoreProperties` to `fetchLogstorePrefixProperties`. WDYT?